composer.json

```diff
@@ -30,7 +30,7 @@
         "gourmet/faker": "~1.0",
         "gourmet/robo": "dev-master",
         "friendsofcake/bootstrap-ui": "*",
-        "friendsofcake/crud": "dev-cake3",
+        "friendsofcake/crud": "dev-master",
```

Comment:
The last tag was a few months ago (approaching 100 commits) so I left it tracking the dev branch.

Let me know if you want to track tagged versions instead and I'll make the change.